packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -45,6 +45,14 @@ const multiSelectComboBox = css`
     flex-basis: 0;
     padding: 0;
   }
+
+  :host([all-chips-visible]) #chips {
+    display: contents;
```

Comment:
This is needed to get `<input>` element wrapping together with chip elements. In case of Goran's addon, it worked out of the box because there wasn't any additional container around chip elements.